### PR TITLE
Homepage tweaks

### LINKED ISSUE #122: [Home page] Add see more to featured project row
- [ ] Add "See more featured projects" arrow link
- [ ] Link to page with highlighted projects filtered

Figma link here: [[TASK] Home - Featured project row update](https://www.figma.com/file/7PSJtPUtwrdPRtXbtQFRFZ/helpwithcovid.com?node-id=503%3A2 )

Screen Shot 2020-04-07 at 10 52 16 PM

Need @cat.liu's approval!

### LINKED ISSUE #126: [Homepage/Project Card] Left aligned project card components when 0 volunteers
Project card components: location and time align left when there are no volunteers. See second project card vs. first one. 

If no volunteers —> “0 volunteers”

9A621736-1834-4061-BDDC-E3F0ECFC1018

### PR DESCRIPTION
**Pull request by caseypugh**
- fixes #126 - Shows `No volunteers` when volunteers=0
- fixes #122 - also adds filtering to project listing (w/ specs)
- touches on #129 
- adding periods `.` to sentences on homepage per feedback from my prof. copywriter partner :) 

image

### CODE REVIEW

File: app/views/projects/_project-card.html.erb [hunk 1]
```diff
@@ -21,9 +21,13 @@
...
     <div class="relative grid grid-cols-3 gap-2 text-sm mt-3">
       <% total_volunteers = project.volunteers.count %>
-      <% if total_volunteers > 0 %>
-        <div class="font-bold"><%= total_volunteers %> <%= 'volunteer'.pluralize total_volunteers %></div>
-      <% end %>
+      <div class="font-bold">
+        <% if total_volunteers > 0 %>
+          <%= total_volunteers %> <%= 'volunteer'.pluralize total_volunteers %>
+        <% else %>
+          No volutneers
```
Comment (sradu) [inline app/views/projects/_project-card.html.erb:28]: typo

File: app/views/projects/_project-card.html.erb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <%= link_to project, class: 'relative block shadow-lg transform transition-transform duration-100 hover:scale-105' do %>
   <div class="w-full bg-center bg-cover" style="height: 130px; background-image: url(<%= project.cover_photo(local_assigns[:group]) %>);"></div>
 
-  <% if project.accepting_volunteers? %>
+  <% if project.accepting_ss? %>
```
Comment (sradu) [inline app/views/projects/_project-card.html.erb:4]: my command key isn't working that well either

Comment (caseypugh) [reply app/views/projects/_project-card.html.erb:4]: oh wow.. lol. i keep on switching between Mac and Windows and forget which key i'm using.

Comment (caseypugh) [reply app/views/projects/_project-card.html.erb:4]: It's kind of strange the tests passed on this.. going to investigate..

**Pull request merged**